src/library/Uno.Material/Styles/Controls/Ripple.xaml

```diff
@@ -0,0 +1,123 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+					xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+					xmlns:controls="using:Uno.Material.Controls">
+
+	<!-- Starting implementation acknowledgement from project https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/blob/2740f14a814896d42032ae0013b765a8a0ec04c3/MaterialDesignThemes.Uwp/Themes/Generic.xaml -->
+
+	<ResourceDictionary.MergedDictionaries>
+		<ResourceDictionary Source="../Application/Colors.xaml" />
+		<ResourceDictionary Source="../Application/SelectionControlColors.xaml" />
+	</ResourceDictionary.MergedDictionaries>
+
+
+	<!-- Known Issue - Wasm pressed state/ripple effect is only applied on first press -->
+	<Style TargetType="controls:Ripple">
+		<Setter Property="HorizontalAlignment"
+				Value="Stretch" />
+		<Setter Property="VerticalAlignment"
+				Value="Stretch" />
+		<Setter Property="Background"
+				Value="Transparent" />
+		<Setter Property="Feedback"
+				Value="{StaticResource BackgroundColor}" />
+		<Setter Property="Template">
+			<Setter.Value>
+				<ControlTemplate TargetType="controls:Ripple">
+					<Grid Background="Transparent"
+						  Margin="{TemplateBinding Margin}"
+						  CornerRadius="{TemplateBinding CornerRadius}">
+						<VisualStateManager.VisualStateGroups>
+							<VisualStateGroup x:Name="CommonStates">
+								<VisualState x:Name="Normal" />
+								<!--Uncomment for hover effect-->
```

Comment:
same here, do we have an explanation?

Reply:
It's nice to have feature but not explicitly needed according the zeplin design